test/fixture/sessionEncryption.js

```diff
@@ -0,0 +1,25 @@
+const { JWK, JWE } = require('jose');
+const hkdf = require('futoin-hkdf');
+
+const deriveKey = (secret) => hkdf(secret, 32, { info: 'JWE CEK', hash: 'SHA-256' });
+const epoch = () => Date.now() / 1000 | 0;
+
+const key = JWK.asKey(deriveKey('__test_secret__'));
+const payload = JSON.stringify({sub: '__test_sub__'});
+
+const encryptOpts = {
+  alg: 'dir',
+  enc: 'A256GCM',
+  zip: 'DEF',
+  uat: epoch(),
+  iat: epoch(),
+  exp: epoch() + ( 7 * 24 * 60 * 60 )
```

Comment:
Given that epoch is a function, is there any chance that there might be a slight diff between these 3 different calls? and if that affects the token, wouldn't it be better to call the function once before defining the `encryptOpts` object?

Another nit one, I'd move `7 * 24 * 60 * 60` to a constant and give it a name, or at least add a comment giving the quick result e.g. `// 1 week in secs`